CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+Version 17.6
+============
+
+* Added Garnet backend so that it can be used in simulations. 
```

Comment:
Not very important, but good to add also link to the pull request, similar to other changelog entries.
```suggestion
* Added Garnet backend so that it can be used in simulations.
  `#156 <https://github.com/iqm-finland/qiskit-on-iqm/pull/156>`_
```